File: 66-plus-one/66-plus-one.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        
        s = ''
        for i in digits:
            s += str(i)

        res1 = int(s) + 1 
        ans1 = []
        while res1 > 0 :
            ld = res1 % 10 
            res1 //= 10
            ans1.append(ld)

        return list(reversed( ans1))        
        
        

File: 66-plus-one/66-plus-one.cpp
class Solution {
public:
    vector<int> plusOne(vector<int>& digits) {
        	int n = digits.size() - 1;
	int x = digits[n];
	if (x != 9)
	{
		digits[n] = x + 1 ;

	}
	else
	{

		while(n >= 0 && digits[n] == 9)
		{
			digits[n] = 0 ;
			n--;
		}

		if (n < 0 )
		{
			digits[n+1] = 1 ;
			digits.push_back(0);
		}
		else
		{
			digits[n] += 1 ;
		}

	}

	return digits;
        
    }
};